lib/net/ldap/connection.rb

```diff
@@ -3,6 +3,9 @@
 class Net::LDAP::Connection #:nodoc:
   include Net::LDAP::Instrumentation
 
+  # Seconds before failing for socket connect timeout
+  DefaultConnectTimeout = 1
```

Comment:
Could we make this value a bit more generous so it doesn't surprise any users with slow connections?